README.md

```diff
@@ -1 +1,39 @@
 # bridge-workers
+
+### Prerequisites
+
+* Install Docker.
+* Two files on the host, `.aws/credentials` and `.bridge/workers.conf`, are shared with the workers container.
+  * Make sure the default AWS credentials profile in `.aws/credentials` has the correct credentials.
+  * Make sure `.bridge/workers.conf` has the `bridge.user` entry and is set to your user name.
```

Comment:
`.aws/credentials` is the default AWS credentials file (see http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-config-files).  The `bridge.user` entry is the only entry required.  Otherwise, the Docker user is "root", which will mess up with the DynamoDB table names.